### PR TITLE
Create a file for Quezon City, Philippines

### PR DESCRIPTION
**Pull request by francisvicta**
This file will add awesome things to do in QC, Ph.

- [ ] Add places to eat
- [ ] Things to do

### CODE REVIEW

File: francisvicta-quezon.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Quezon City, Philippines
```
Comment (francisco-cho) [inline francisvicta-quezon.md]: how do I pronounce this?

Comment (francisvicta) [inline francisvicta-quezon.md]: @francisco-cho keson

Comment (caribmorpho) [inline francisvicta-quezon.md]: Is Quezon City part of Metro Manila?

Comment (francisvicta) [inline francisvicta-quezon.md]: @caribmorpho It is. It is a little confusing, but there is also a city called Manila, which is part of Metro Manila.

**Pull request merged**